app/[locale]/error.tsx

```diff
@@ -15,4 +20,23 @@ const ErrorPage: FC<{ error: Error }> = ({ error }) => {
   );
 };
 
-export default ErrorPage;
+const ErrorPage: FC<{ error: Error }> = ({ error }) => {
+  useMemo(() => captureException(error), [error]);
+
+  return (
+    <CenteredLayout>
+      <div className="glowingBackdrop" />
+
+      <main>
+        500
+        <h1 className="special -mt-4">Internal Server Error</h1>
```

Comment:
Should we translate this?

Reply:
would the app be able to display the i18n key in the event of an error?

Reply:
Yes. If the error is unrelated to i18n, but if by any chance the error is with i18n, well, then it will fallback to `global-error.tsx` I assume.

In general the `error.tsx` is triggered when there's client-side error, and our i18n happens on the server-side (also on client-side as being propagated) but the error page is more like runtime client-side errors.